roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +86,50 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  become: true
+  tags:
+    - configure
+    - nova
+  ansible.posix.synchronize:
+    src: "{{ edpm_nova_config_sync_src }}"
+    dest: "{{ edpm_nova_config_dest }}"
+    recursive: true
+    delete: true
+    mode: "push"
```

Comment:
i belive push is the default by the way but that fine.

Reply:
yes you are right, done